examples/fft_physical_x/Makefile

```diff
@@ -8,29 +8,34 @@ LIBS = -L../../ -l$(LIBDECOMP) $(LIBFFT) $(LFLAGS)
 NP ?= 1
 MPIRUN ?= mpirun
 
-all: fft_physical_x fft_grid_x
+all: fft_c2c_x fft_r2c_x fft_grid_x
 
-fft_physical_x: fft_physical_x.o
+fft_c2c_x: fft_c2c_x.o
+	$(FC) $(FFLAGS) $(OPT) $(DEFS) $(INC) -o $@ $< $(OBJ) $(LIBS)
+
+fft_r2c_x: fft_r2c_x.o
 	$(FC) $(FFLAGS) $(OPT) $(DEFS) $(INC) -o $@ $< $(OBJ) $(LIBS)
 
 fft_grid_x: fft_grid_x.o
 	$(FC) $(FFLAGS) $(OPT) $(DEFS) $(INC) -o $@ $< $(OBJ) $(LIBS)
 
 ifeq ($(PARAMOD),gpu)
 check:
-	$(MPIRUN) -n $(NP) ./bind.sh ./fft_physical_x
+	$(MPIRUN) -n $(NP) ./bind.sh ./fft_c2c_x
+	$(MPIRUN) -n $(NP) ./bind.sh ./fft_r2c_x
 	$(MPIRUN) -n $(NP) ./bind.sh ./fft_grid_x
 else
 check:
-	$(MPIRUN) -n $(NP) ./fft_physical_x
+	$(MPIRUN) -n $(NP) ./fft_c2c_x
+	$(MPIRUN) -n $(NP) ./fft_r2c_x
 	$(MPIRUN) -n $(NP) ./fft_grid_x
 endif
 
 mem_leak:
-	valgrind --leak-check=full --show-leak-kinds=all $(MPIRUN) -n 1 ./fft_physical_x 1 1
+	valgrind --leak-check=full --show-leak-kinds=all $(MPIRUN) -n 1 ./fft_c2c_x 1 1
```

Comment:
Do we want to force a serial run here or is it worth using ```-n $(NP)``` (and changing 1s to 0s at the end, assuming they are prow/pcol)?

Reply:
If running in parallel would be something like
```
valgrind ... --log-file=valgr.log%p $(MPIRUN) ...
```

Reply:
we do not want to force a serial run!

Reply:
Right now we force serial run when running valgrind and hunting memory leaks. I am not sure about the syntax for running valgrind in parallel.

Reply:
If this is an issue, this is out of the scope of the present PR